source/css/sass/patterns/molecules/article-type.scss

```diff
@@ -0,0 +1,21 @@
+@import "../../mixins/types";
+@import "../../variables/baselinegrid";
+@import "../../variables/color";
+@import "../../variables/font";
+@import "../../mixins/spacing";
+
+.article__type {
```

Comment:
Appears unused?

Reply:
I have deleted this css file as no longer needed.